.github/workflows/plugins.yml

```diff
@@ -643,7 +643,8 @@ jobs:
     strategy:
       matrix:
         node-version: [14, 16]
-        range: ['>=9.5 <11.1', '>=11.1 <13.2', '>=13.2']
+        # testing 10.2 in isolation due to a bug in the test suite
+        range: ['10.2', '>=11.1 <13.2', '>=13.2']
```

Comment:
Should the range for 10.x be a bit wider? Or is 10.2 the only v10?

Reply:
Which I now realize means that my comment is unnecessary

Reply:
Comment has been removed

Reply:
Actually, this should definitely be `>=10.2 <11.1`, otherwise 11.0 will no longer be supported.

Reply:
Should the `10.2` not be `>=10.2 <11.1` to match the hook? Is 10.2 the last 10.x release? Seems a bit odd to me to support a single specific release out of a whole major range. 🤔